extensions/wikia/TriviaQuizzes/js/ext.TriviaQuizzes.js

```diff
@@ -0,0 +1,26 @@
+require(['jquery', 'content_types_consumption', 'wikia.trackingOptIn'], function ($, consumption, trackingOptIn) {
+	'use strict';
+	$(function() {
+		$('.WikiaRail').on('afterLoad.rail', function() {
+			// For now: if they are IE11, remove the entire module until we have better support
+			if($.browser.msie) {
+				$('#wikia-recent-activity').remove();
+			} else {
+				var cityId = window.wgCityId ? window.wgCityId : 0;
```

Comment:
In newer versions of MediaWiki, it is preferred to access global JS vars via `mw.config.get`, i.e. `mw.config.get('wgCityId');`. You can require the `mw` module for this.

Reply:
Fixed in 88f7f44be9d